CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
 # Version changelog
 
+## 0.31.0
+
+* Update actions/setup-go to v5 ([#784](https://github.com/databricks/databricks-sdk-go/pull/784)).
+* Retry update of catalog in test to avoid flakiness ([#788](https://github.com/databricks/databricks-sdk-go/pull/788)).
+* Support creating a new workspace client from an account client ([#792](https://github.com/databricks/databricks-sdk-go/pull/792)).
```

Comment:
This should be clearly advertised, and we can even include the example from the change in the changelog.